style.css

```
@@ -0,0 +1,117 @@
+
+.big {
+    text-align: left;
+    width: 360px;
+    height: 600px;
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    border-bottom: 1px solid grey;
+    border-radius: 9px;
+    padding: 15px;
+    margin:auto;
+}
+
+.today {
+    color: grey;
+    text-align: center;
+    padding: 5px;
+    margin-top: 5px;
+    font-size: 10px;
+}
+
+.TodoList {
+    text-align: center;
+    padding: 10px;
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 10px;
+    color: rgb(56, 66, 40);
+}
+
+.input {
+    height: 25px;
+    margin-bottom: 10px;
+    width: 280px;
+    border-radius: 12rem;
+    border-color: rgb(210, 210, 210);
+    padding-left: 10px;
+}
+
+.addButton {
+    width: 50px;
+    height: 31px;
+    border-radius: 100rem; 
+    border-color: #449a80;
+    background-color: #449a80;
```

메인컬러처럼 자주 쓰이는 색상들은 css 변수를 통해 관리하면 개발하기도 유지보수하기도 더욱 편할 것 같습니다!!

```
:root {
  --main-blue: #449a80;
}
```

```suggestion
    border-color: var(--main-blue);
    background-color: var(--main-blue);
```

---

이 방법은 몰랐는데 알려주셔서 감사합니다!